src/lib.rs

```diff
@@ -1363,6 +1363,12 @@ impl Build {
                         target
                     ));
                 }
+            } else if target.contains("wasm32") && target.contains("linux") {
```

Comment:
Would it be better to do an exact match?

```suggestion
            } else if target == "wasm32-wali-linux-musl" {
```

Reply:
I'd prefer to avoid `.contains`, and use `target.arch == "wasm32" && target.os == "linux"` (or similar)